controllers/pulp/controller_test.go

```diff
@@ -166,19 +166,19 @@ var _ = Describe("Pulp controller", Ordered, func() {
 	}
 
 	envVarsApi := []corev1.EnvVar{
-		{Name: "POSTGRES_SERVICE_HOST", Value: PulpName + "-database-svc"},
-		{Name: "POSTGRES_SERVICE_PORT", Value: "5432"},
 		{Name: "PULP_GUNICORN_TIMEOUT", Value: strconv.Itoa(90)},
 		{Name: "PULP_API_WORKERS", Value: strconv.Itoa(2)},
+		{Name: "POSTGRES_SERVICE_HOST", Value: PulpName + "-database-svc"},
+		{Name: "POSTGRES_SERVICE_PORT", Value: "5432"},
 		{Name: "REDIS_SERVICE_HOST", Value: PulpName + "-redis-svc." + PulpNamespace},
 		{Name: "REDIS_SERVICE_PORT", Value: strconv.Itoa(6379)},
 	}
 
 	envVarsContent := []corev1.EnvVar{
-		{Name: "POSTGRES_SERVICE_HOST", Value: PulpName + "-database-svc"},
-		{Name: "POSTGRES_SERVICE_PORT", Value: "5432"},
 		{Name: "PULP_GUNICORN_TIMEOUT", Value: strconv.Itoa(90)},
 		{Name: "PULP_CONTENT_WORKERS", Value: strconv.Itoa(2)},
```

Comment:
not related to this PR, but this timeout and number of workers should come from the CR spec

Reply:
ignore this, I didn't realize it was a test